apps/web/tsconfig.json

```diff
@@ -6,78 +6,30 @@
     "emitDeclarationOnly": false,
     "noEmit": true,
     "paths": {
-      "~/abis/*": [
```

Comment:
I'm pretty sure Moon has stopped formatting these files, which puts prettier back in control. This has happened on a few PRs now